tango_simlib/tests/test_sim_xmi_parser.py

```diff
@@ -224,12 +228,25 @@ def test_attribute_list(self):
         """ Test whether the attributes specified in the POGO generated xmi file
         are added to the TANGO device
         """
+        # First testing that the attribute with data format "IMAGE" is not in the device.
+        attribute_name = 'image1'
+        device_attributes = set(self.device.get_attribute_list())
+        self.assertNotIn(attribute_name, device_attributes,
+                         "The attribute {} has been added to the device.".
+                         format(attribute_name))
+        not_added_attr = self.device.read_attribute('AttributesNotAdded')
+        not_added_attr_names = getattr(not_added_attr, 'value')
```

Comment:
Do we really need to use `getattr()` here?  We always know the name of the attribute, so isn't `not_added_attr.value` sufficient?  Same in the other test file.